pages.bn/windows/color.md

```diff
@@ -0,0 +1,16 @@
+# color
+
+> কনসোল পূর্বপরক্ষ এবং পৃষ্ঠভূমির রঙ নির্ধারণ করুন।
+> আরও তথ্য পাবেন: <https://learn.microsoft.com/windows-server/administration/windows-commands/color>.
```

Comment:
```suggestion
> আরও তথ্য পাবেন: <https://learn.microsoft.com/windows-server/administration/windows-commands/color>।
```
I am working on standardizing the period back to the Native one (as we do the same in Hindi) in #11331